swarm/pss/pss_test.go

```diff
@@ -475,6 +482,196 @@ func testAsymSend(t *testing.T) {
 	}
 }
 
+// params in run name:
+// nodes/msgs/addrbytes/adaptertype
+// if adaptertype is exec uses execadapter, simadapter otherwise
+func TestNetwork(t *testing.T) {
+	t.Run("8/1024/2/sim", testNetwork)
+}
+
+func testNetwork(t *testing.T) {
```

Comment:
Given the size of this test, it would be useful to describe what it is testing in a comment.